src/status_im/chat/handlers.cljs

```diff
@@ -392,10 +392,9 @@
 (defn delete-chat!
   [_ [_ chat-id]]
   (let [{:keys [debug? group-chat]} (chats/get-by-id chat-id)]
-    (when group-chat
-      (if debug?
-        (chats/delete chat-id)
-        (chats/set-inactive chat-id)))))
+    (when (and (not debug?) group-chat)
```

Comment:
As I understand this condition means that only group chats are removed (and inactivated at the same time) when `debug?` is `false`.